opentech/apply/users/management/commands/migrate_users.py

```diff
@@ -78,3 +78,14 @@ def get_user_groups(self, user):
                 groups.append(self.groups.filter(name=group_name).first())
 
         return groups
+
+    def get_email(self, user, anonymize=False):
+        email = user['mail']
+        if not anonymize or self.is_staff(email):
+            return email
+
+        return "aeon+%s@torchbox.com" % user['uid']
```

Comment:
prefer f string of format. `%` is old style.

Reply:
didn't want an extra assignment :)